cfme/utils/ssh.py

```diff
@@ -380,11 +379,6 @@ def write_output(line, file):
             if exit_status != 0:
                 logger.warning('Exit code %d!', exit_status)
             return SSHResult(rc=exit_status, output=''.join(output), command=command)
-        except paramiko.SSHException:
```

Comment:
Q: why we are removing `paramiko` exception? I think better to handle both as the session comes from paramiko.

Reply:
same as above, but in short... when `paramiko.SSHException` is raised, we will try to reconnect rather then just cover the exception under the carpet and then return the misleading `SSHResult` with `rc=1`. For example: When SSH failed to connect, we didn't run any command, thus returning `SSHResult` with rc is just wrong.